dockerclient.go

```diff
@@ -258,6 +258,37 @@ func (client *DockerClient) ContainerChanges(id string) ([]*ContainerChanges, er
 	return changes, nil
 }
 
+func (client *DockerClient) ContainerStats(id string, stopChan <-chan struct{}) (<-chan StatsOrError, error) {
+	uri := fmt.Sprintf("/%s/containers/%s/stats", APIVersion, id)
```

Comment:
Unfortunately stats don't get added until API 1.17, so you'll have to bump APIVersion.  I don't know what the delta between 1.15 and 1.17 is though and if we'd be missing anything here.